docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+    "\n",
+    "We can also create a [ParticleList] by adding [Particle] and/or [CustomParticle] objects together."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f69c8ce6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proton + electron + custom_particle"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df2415de",
+   "metadata": {},
+   "source": [
+    "## Dimensionless particles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1af4fc4",
+   "metadata": {},
+   "source": [
+    "[DimensionlessParticle]: ../../api/plasmapy.particles.particle_class.DimensionlessParticle.rst\n",
+    "\n",
+    "When we need a dimensionless representation of a particle, we can use the [DimensionlessParticle] class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f57a0f64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dimensionless_particle = DimensionlessParticle(mass=0.000545, charge=-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf598cb0",
+   "metadata": {},
+   "source": [
+    "The properties of dimensionless particles may be accessed by its attributes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7b0f9755",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dimensionless_particle.mass"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4920b68",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dimensionless_particle.charge"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29e23e5f",
+   "metadata": {},
+   "source": [
+    "[DimensionlessParticle]: ../../api/plasmapy.particles.particle_class.DimensionlessParticle.rst\n",
+    "[ParticleList]: ../../api/plasmapy.particles.particle_collections.ParticleList.rst\n",
+    "\n",
+    "Because a [DimensionlessParticle] does not uniquely describe a physical particle, it cannot be contained in a [ParticleList].  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "efe199c8",
+   "metadata": {},
+   "source": [
+    "## Nuclear reactions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58fb006b",
+   "metadata": {},
+   "source": [
+    "[plasmapy.particles]: ../../particles/index.rst\n",
+    "\n",
+    "We can use [plasmapy.particles] to calculate the energy of a nuclear reaction using the `>` operator.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d8a9ab9e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deuteron = Particle(\"D+\")\n",
+    "triton = Particle(\"T+\")\n",
+    "alpha = Particle(\"α\")\n",
+    "neutron = Particle(\"n\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb95c644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "energy = deuteron + triton > alpha + neutron"
```

Comment:
Can you add an example of an invalid reaction to show the excellent exception messages you have for this? I think that's a big educational feature too.

Reply:
Good idea!